### ISSUE TITLE
Disable fan control leads to "webman not ready"

### ISSUE THREAD

**Issue by Rast1234** [aldostools/webMAN-MOD#311]
Hi. My version is `1.47.26` on freshly installed `Rebug 4.85.1 LITE` on a blank HDD.

If i change fan settings to disable dynamic fan control (which selects `SYSCON`), and then reboot, webman does not load. Games don't mount with **"webman not ready"** message, FTP and HTTP services are not accessible.

To recover from this i launch multiman, delete `/dev_hdd0/tmp/wm_config.bin` by FTP and reboot. 

I've seen fan control related issues fixed recently ( #227 #228 #233 ), maybe it accidentally caused some regress?

UPD: tested with `1.47.24`, still the same. Guess it's my FW version then?

**Comment by Rast1234** [comment]
Installed rebug 4.84.2, no issues. it's caused by FW upgrade then. Close issue if you want.

**Comment by aldostools** [comment]
Thank you for your feedback. I also stayed on 4.84.2 because it is more stable.

**Comment by Rast1234** [comment]
I got back to my PS3, and it happens again: last version of WM; Rebug 4.84.2 REX; fan settings on SYSCON. Can i help you with debugging this? Maybe collect crashdump somehow or provide my wm_config.bin?
Seems i can't reopen the issue, so i will tag you in case you don't see comments on closed issues... @aldostools

**Comment by aldostools** [comment]
Try enabling dynamic fan control. That's the option that use most of the users.

**Comment by Rast1234** [comment]
Yeah, default option works, but after playing with different options i decided to stay with default PS3's fan profile (yes i am aware of consequences of overheating). Will use dynamic for now anyway :)

**Comment by Joonie86** [comment]
No further feedback
